src/main/flight/mixer.c

```diff
@@ -976,3 +970,26 @@ void filterServos(void)
 #endif
 }
 
+uint16_t pwmToMotor(uint16_t pwm)
+{
+    uint16_t motor = pwm;
+#ifdef USE_DSHOT
+    if (masterConfig.motorConfig.motorPwmProtocol == PWM_TYPE_DSHOT150 || masterConfig.motorConfig.motorPwmProtocol == PWM_TYPE_DSHOT600) {
+        motor = pwm == 1000 ? 0 : constrain((pwm - 977) * 2, 48, 2047);
```

Comment:
No it won't solve it, this is the conversion that is used for the configurator to FC direction. But setting the ESC to disarmed for values < 1000 can't hurt.

977 is the offset used for the conversion [1001, 2000] to [48, 2047], so it is 1001 - (48 / 2). I didn't think putting it in as the formula would increase the readability, but I am open for discussion on that.